themes/qgrid/source/css/layout-page.scss

```diff
@@ -0,0 +1,104 @@
+@import "variables";
```

Comment:
what does layout-page mean? name is not obvious, if this is only for pages please call it `page.scss`